src/lib/markbind/src/handlers/cyclicReferenceError.js

```diff
@@ -0,0 +1,20 @@
+const MAX_RECURSIVE_DEPTH = 200;
+const ERROR_MESSAGE = 'Cyclic reference detected.';
+
+class CyclicReferenceError extends Error {
+  static get MAX_RECURSIVE_DEPTH() {
+    return MAX_RECURSIVE_DEPTH;
+  }
+
+  static get ERROR_MESSAGE() {
+    return ERROR_MESSAGE;
```

Comment:
This static class variable is not used. We can remove it. Alternatively we could treat it as static class variable declaration like: 
```suggestion
    return 'Cyclic reference detected.;
```

And use `CyclicReferenceError.ERROR_MESSAGE` in the constructor.

Reply:
Hmm, I was of the impression that we should declare the ERROR_MESSAGE like a constant, i.e. `const ERROR_MESSAGE = ...`
But I guess, since we're only going to use it once, it doesn't make too much sense to declare it this way.
Removed. Thanks!